package.json

```diff
@@ -60,25 +60,25 @@
     "video.js": "^5.18.0 || ^6 || ^7"
   },
   "devDependencies": {
+    "@gkatsev/rollup-plugin-node-builtins": "^2.1.3",
     "conventional-changelog-cli": "^2.0.1",
-    "conventional-changelog-videojs": "^3.0.0",
+    "conventional-changelog-videojs": "^3.0.1",
     "doctoc": "^1.3.1",
     "husky": "^1.0.0-rc.13",
-    "jsdoc": "https://github.com/BrandonOCasey/jsdoc#feat/plugin-from-cli",
-    "karma": "^3.0.0",
+    "jsdoc": "3.6.6",
+    "karma": "^6.1.1",
     "lint-staged": "^7.2.2",
     "not-prerelease": "^1.0.1",
-    "npm-merge-driver-install": "^1.0.0",
+    "npm-merge-driver-install": "^2.0.1",
     "npm-run-all": "^4.1.5",
-    "pkg-ok": "^2.2.0",
+    "pkg-ok": "^2.3.1",
     "rollup": "^0.66.0",
-    "rollup-plugin-node-builtins": "^2.1.2",
     "rollup-plugin-node-globals": "^1.4.0",
     "shx": "^0.3.2",
     "sinon": "^6.1.5",
     "videojs-generate-karma-config": "^5.0.1",
     "videojs-generate-rollup-config": "~2.3.0",
```

Comment:
Did you want to try to update the karma/rollup config as well? or was an update not working

Reply:
I was trying to do the most minimal change without needing to potentially update the configs themselves.

Reply:
That's fine, we can do it another time if it comes up